### PR TITLE
feat(KUI-1267): upgrade to KTH Style 10

### PR DESCRIPTION
**Pull request by karlandindrakryggen**
This PR upgrades KTH Style to version 10 and includes some more cleanup and fixes:

- Upgrade to KTH style packages 
  - Includes major cleanup of CSS
  - Adding same shared concept as in public web apps with copied files in `_shared.scss` and `components-shared` folder 
- Upgrade of other dependencies (minor/patch)
- Hardcoded toolbar URL are replaced with env variable
- Favicon path is fixed

https://kth-se.atlassian.net/browse/KUI-1267

### CODE REVIEW

**Comment by karlandindrakryggen** [comment]
Related PR in kursinfo-admin-web: https://github.com/KTH/kursinfo-admin-web/pull/273

**Review by amirhossein-haerian — APPROVED** [review]

**Review by axelbjo — APPROVED** [review]

File: public/css/_shared.scss [hunk 1]
```diff
@@ -0,0 +1,166 @@
+// This file contains style shared (copied) between the admin apps of Om kursen apps (kurs-pm-admin-web, kurs-pm-data-admin-web, kursinfo-admin-web, kursutveckling-admin-web)
```
Comment (belanglos) [inline public/css/_shared.scss:1]: Same comment here. Link to confluence?

File: public/js/app/components-shared/Alert.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/**
+ * https://intra.kth.se/style/en/components/alert
+ *
+ * Changelog:
+ * - 2024-04- - initial version (release of kth-style 10 public apps)
```
Comment (belanglos) [inline public/js/app/components-shared/Alert.jsx:5]: Day?

File: public/js/app/views/AdminPage.jsx [hunk 3]
```diff
@@ -423,142 +417,111 @@ function AdminPage() {
...
-              <Row key="success-alert" className="w-100 my-0 mx-auto upper-alert">
-                <Alert color="success">{alertSuccess} </Alert>
-              </Row>
-            )}
-            {state.alertError.length > 0 && (
-              <Row key="error-alert" className="w-100 my-0 mx-auto upper-alert">
-                <Alert color="danger">{state.alertError} </Alert>
-              </Row>
-            )}
+            {state.alert.length > 0 && <Alert type="info">{state.alert} </Alert>}
```
Comment (belanglos) [inline public/js/app/views/AdminPage.jsx:443]: 😍

**Review by belanglos — APPROVED** [review]
Horse job also here!
Great work! 💯 

Basically same comments as in kursinfo-admin-web.

**Pull request merged**